aiopvapi/hub.py

```diff
@@ -61,13 +82,36 @@ def __eq__(self, other):
 
 
 class Hub(ApiBase):
-    api_path = "api"
-
     def __init__(self, request):
-        super().__init__(request, self.api_path)
+        api_version = request.api_version
+        if api_version >= 3:
```

Comment:
This block is repeated quite a bit, and we also have the `initial_api_str` method, this feels DRY violating, can (should?) we tidy this up a bit?

Reply:
Might depend if we want to have separate versions of Hub per API level.